node/src/service.rs

```diff
@@ -433,7 +408,7 @@ fn start_consensus(
         relay_chain_slot_duration,
         proposer,
         collator_service,
-        authoring_duration: Duration::from_millis(1500),
+        authoring_duration: Duration::from_millis(2000),
```

Comment:
Any reason why this increased to 2000 against the recommendation here: https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-2---update-parachain-nodes?

Reply:
https://github.com/paritytech/polkadot-sdk/blob/d5fe478e4fe2d62b0800888ae77b00ff0ba28b28/templates/parachain/node/src/service.rs#L214